### PR TITLE
Revise WordCount Example Walkthrough

### PR DESCRIPTION
**Pull request by tgroh**
Fiddle with monospace markers.

Update code snippets.

Reorder logging example and update "[Google ]Cloud Logging" ->
Stackdriver Logging

### CODE REVIEW

**Comment by tgroh** [comment]
R: @melap @davorbonaci

File: src/get-started/wordcount-example.md [hunk 1]
```diff
@@ -169,7 +171,7 @@ Note that the `Write` transform produces a trivial result value of type `PDone`,
 Run the pipeline by calling the `run` method, which sends your pipeline to be executed by the pipeline runner that you specified when you created your pipeline.
 
 ```java
-p.run().waitUntilFinish();
+p.run();
```
Comment (davorbonaci) [inline src/get-started/wordcount-example.md]: revert

File: src/get-started/wordcount-example.md [hunk 2]
```diff
@@ -456,7 +467,7 @@ This feature is not yet available in the Beam SDK for Python.
 
 ### Reusing PTransforms over windowed PCollections
 
-You can reuse existing `PTransform`s, that were created for manipulating simple `PCollection`s, over windowed `PCollection`s as well.
+You can reuse existing `PTransforms` that were created for manipulating simple `PCollections` over windowed `PCollections` as well.
```
Comment (davorbonaci) [inline src/get-started/wordcount-example.md]: not sure about this formatting

File: src/get-started/wordcount-example.md (hunk 1 above)
Comment (tgroh) [inline src/get-started/wordcount-example.md]: done.

File: src/get-started/wordcount-example.md (hunk 2 above)
Comment (tgroh) [inline src/get-started/wordcount-example.md]: rmd code ticks

**Pull request closed without merging**